packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -280,8 +289,16 @@ export class AppManagementClient implements DeveloperPlatformClient {
       )
   }
 
-  async templateSpecifications(_appId: string): Promise<ExtensionTemplate[]> {
-    return stubbedExtensionTemplates()
+  async templateSpecifications({organizationId}: MinimalAppIdentifiers): Promise<ExtensionTemplate[]> {
+    const allExtensions = await Promise.all(templateJsonUrls.map(async (url) => {
+      const response = await fetch(url)
+      return response.json() as Promise<GatedExtensionTemplate[]>
```

Comment:
We should handle errors here given we're taking a new dependency on github's asset server.

Reply:
<img width="1177" alt="Screenshot 2024-05-31 at 0 04 25" src="https://github.com/Shopify/cli/assets/6288426/c9177d32-a3ac-45eb-9261-f43bf1b9c735">